### DISCUSSION TITLE
Suspicious connection

### DISCUSSION THREAD

**Discussion by IToubol** [community/community#107462] [New to GitHub]
Hello.

I received an email from GitHub informing me that a connection to my account was established from a location in Israel (I live in France).
I don't understand what happened and I'm worried: how was this connection possible?
Is there a security breach in my account?

I indicate that I shared on an Israelian forum a link to a streamlit application which is based on code posted on my Github account,
but I do not think that this opens the door to an external connection to my account.

Thanks for your response.

**Comment by Olayiwolaaa** [comment]
Regardless, the how isn't important if you loose your account. What you need is to do is fortify your account.
- Change your password to a strong and unique one.
- Enable 2FA for more security
- Review recent activity and revoke access for any unauthorized apps
- check repo access permissions and limit them to trusted collaborators
- and avoid clicking suspicious links man.

## Stay safe online!
# Mark as answer if this was helpful.
